settings.gradle

```diff
@@ -4,6 +4,7 @@ include 'auth-service'
 include 'app-shell'
 include 'gateway-service'
 include 'cart'
-include 'history'
+include 'history:frontend'
+include 'history:server'
```

Comment:
Can this just include main build file, which will include these two? Top-level build shouldn't be aware of MF builds